config.yml

```diff
@@ -159,20 +159,20 @@ params:
   # For help, see the wiki:
   # https://github.com/cstate/cstate/wiki/Customization
   systems:
-    - name: Wallabag.it homepage
+    - name: wallabag.it homepage
       displayName: Homepage
       description: Presentation of wallabag.it service
       link: https://www.wallabag.it
-      category: Wallabag.it
-    - name: Wallabag.it service
+      category: wallabag.it
+    - name: wallabag.it service
```

Comment:
Si tu changes le nom d’un `system`, il faut que tu le changes aussi dans les tickets qui l’utilisent, comme `content/issues/2025_03_28_maintenance-on-the-wallabagit-service.md`